fs/ggml/type.go

```diff
@@ -12,26 +12,26 @@ type FileType uint32
 const (
 	FileTypeF32 FileType = iota
 	FileTypeF16
-	FileTypeQ4_0
-	FileTypeQ4_1
+	fileTypeQ4_0     // not supported by ollama
```

Comment:
While the quantization has less right now I wouldn't say unsupported since the kernels are still there and importing works